src/plotman/plotman.py

```diff
@@ -163,7 +164,10 @@ def main():
 
         # Status report
         if args.cmd == 'status':
-            print(reporting.status_report(jobs, get_term_width()))
+            print("{0}\n\nUpdated at: {1}".format(
+                reporting.status_report(jobs, get_term_width()),
+                datetime.datetime.today().strftime("%x %X")
```

Comment:
How do you feel about ISO 8601?

```
2021-05-15T00:42:42.716236-04:00
```

```suggestion
                pendulum.now().isoformat(timespec='microseconds')
```

Or possibly full on match the log file name format.  Though, this will rarely exactly match a log so isn't obviously a right choice (and if it were the code should go in a common place).

```suggestion
                pendulum.now().isoformat(timespec='microseconds').replace(':', '_')
```

I would probably tend towards the first.

Reply:
The intention was to format it in a easy readable format, that's why `%x %X` was used, as it would supposedly display date and time in the friendly user's locale .

Reply:
Just adding a couple more examples here as reference.

```console
$ LC_ALL=en_US.UTF-8 python3 -c 'import locale; import datetime; locale.setlocale(locale.LC_ALL, ""); print(datetime.datetime.today().strftime("%x %X"))'
05/15/2021 03:02:02 PM
```
```console
$ LC_ALL=de_DE.UTF-8 python3 -c 'import locale; import datetime; locale.setlocale(locale.LC_ALL, ""); print(datetime.datetime.today().strftime("%x %X"))'
15.05.2021 15:02:09
```

So I personally would prefer ISO 8601 but I guess the obvious response is 'well set your locale to a thing that gives you that!'  But, the existing format string couldn't even theoretically do that.  This probably doesn't matter and I'm likely just indulging my curiosity at this point...  but, there is [`%c`](https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes) that let's the entirety of the string be locale dependent instead of prescribing the date and the time as space separated etc.

```console
$ LC_ALL=de_DE.UTF-8 python3 -c 'import locale; import datetime; locale.setlocale(locale.LC_ALL, ""); print(datetime.datetime.today().strftime("%c"))'
Sa 15 Mai 2021 15:06:31 
```
```console
$ LC_ALL=en_US.UTF-8 python3 -c 'import locale; import datetime; locale.setlocale(locale.LC_ALL, ""); print(datetime.datetime.today().strftime("%c"))'
Sat 15 May 2021 03:06:39 PM
```

Alrighty...  I think I've pushed enough on this.  Need to get my opinions in check.  Let's just get this through.  I'll check out the conflicts etc and get this merged.  Thanks.  `:]`